content/src/Blip.Api.Template/Controllers/ValidateController.cs

```diff
@@ -0,0 +1,298 @@
+                var isValid = IsEmailValid(email);
+
+                return StatusCode(StatusCodes.Status200OK, new ValidateBaseResponse { IsValid = isValid, Result = email });
+
+            }
+            catch (Exception e)
+            {
+
+            }
+
+            return StatusCode(StatusCodes.Status500InternalServerError);
+
+        }
+
+        /// <summary>
+        /// Checks if the current date and time is within sometime sent
+        /// </summary>
+        /// <param name="request">
+        ///     Array of date and begin and end hour to check
+        /// </param>
+        /// <returns>Boolean indicating if now is a valid time and the current datetime as a string</returns>
+        [HttpPost, Route("timeInterval/date")]
+        public IActionResult IsInDateTimeInterval([FromBody] DateInterval[] request)
+        {
+            try
+            {
+                var result = _dateValidator.IsInInterval(request);
+                return StatusCode(StatusCodes.Status200OK, new GenericBaseResponse() { Result = result.ToString().ToLower() });
+            }
+            catch (Exception e)
+            {
+                return StatusCode(StatusCodes.Status500InternalServerError, e);
+            }
+        }
+
+        /// <summary>
+        /// Checks whether the current time is within days and the time interval sent
+        /// </summary>
+        /// <param name="request">
+        ///    Object of days and begin and end hour to check 
+        /// </param>
+        /// <returns>Boolean indicating if there was a valid time and current datetime as a string</returns>
+        [HttpPost, Route("timeInterval/week")]
+        public IActionResult IsInDateTimeInterval([FromBody] DaysOfWeekDateInterval[] request)
+        {
+            try
+            {
+                var result = _dateValidator.IsInInterval(request);
+                return StatusCode(StatusCodes.Status200OK, new GenericBaseResponse() { Result = result.ToString().ToLower() });
+            }
+            catch (Exception e)
+            {
+                return StatusCode(StatusCodes.Status500InternalServerError, e);
+            }
+        }
+
+        private bool IsEmailValid(string text)
+        {
+            try
+            {
+                var mail = new System.Net.Mail.MailAddress(text);
+
+                return true;
+            }
+            catch (Exception e)
+            {
+                return false;
+            }
+        }
+
+        private string TryToExtractName(string text)
+        {
+            var cleanNamePattern = _settings.NameConfig.CleanNameRegexPattern;
+            
+            try
+            {
+                var inputArr = text.Split(new string[] { " " }, StringSplitOptions.RemoveEmptyEntries);
+
+                var inputTreated = string.Empty;
+
+                if (inputArr.Length > 0)
+                {
+                    var inputWithJustLetters = new StringBuilder();
```

Comment:
declare the `StringBuilder` with a neutral name, outside the `if` clause and reuse it after calling the `Clear` method instead of creating a new one in line 276